Module2-Evaluations.js

```diff
@@ -667,41 +541,6 @@ function populateMonthSheetWithEvaluators() {
   }
 }
 
-/**
- * Function to reprocess all evaluation forms within the evaluation window
- */
-function batchProcessEvaluationResponses() {
```

Comment:
Why was this removed? It's very useful to re-process an entire batch of responses, and it should be safe to do so.

Reply:
added back in pr-22 branch